frontend/src/app/projects/models/model-wrapper/model-wrapper.component.ts

```diff
@@ -40,10 +43,15 @@ export class ModelWrapperComponent implements OnInit, OnDestroy {
           this.modelService._model.next(undefined);
         },
       });
+
+    this.breadcrumbSubscription = this.modelService._model.subscribe((model) =>
+      this.breadcrumbService.updatePlaceholder({ model })
+    );
   }
 
   ngOnDestroy(): void {
     this.subscription?.unsubscribe();
+    this.breadcrumbSubscription?.unsubscribe();
```

Comment:
```suggestion
    this.breadcrumbSubscription?.unsubscribe();
```

Unsubscribing is not enough, you also need to reset the placeholder.